packages/integration-sdk-runtime/src/execution/jobState.ts

```diff
@@ -252,7 +252,8 @@ export function createStepJobState({
     },
     addRelationships,
 
-    findEntity: async (_key: string) => {
+    findEntity: async (_key: string | undefined) => {
+      if (!_key) return null;
```

Comment:
Is `null` correct or should it be `undefined`?

Reply:
`null` is correct per the interface:
```diff
-  findEntity: (_key: string) => Promise<Entity | null>;
+  findEntity: (_key: string | undefined) => Promise<Entity | null>;
```
https://github.com/JupiterOne/sdk/pull/573/files#diff-3c56620e866ab752465a4ee3067c23ba05f05c7dab4bea3cc083b4ef2548d135R84